e2e/ratelimit_test.go

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright 2025 NASD Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_test
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"cosmossdk.io/math"
+	"github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/stretchr/testify/require"
+
+	"github.com/noble-assets/noble/e2e"
+)
+
+func TestRateLimit(t *testing.T) {
```

Comment:
it would be nice to have assertions in the test case that ensures forwarding fails too if exceeding the threshold just to ensure we have regression tests in place that ensure the threshold can never be exceeded

Reply:
After testing and chatting internally, we deemed extra forwarding related tests unnecessary. The inflows and outflows of a forward cancel each other out.

For example, consider the tx forward flow: `chainA -> Noble -> chainB`.
Sending from `chainA -> Noble` will cause the threshold to go below the limit. Then, as the forward completes, `Noble -> chainB`, the threshold is met again.
Ultimately, the forward should (in nearly all cases) succeed.